docs/getting-started/security-ui.asciidoc

```diff
@@ -0,0 +1,104 @@
+
+The {security-app} is a highly interactive workspace designed for security
+analysts. It provides a clear overview of events and alerts from your
+environment, and you can use the interactive UI to drill down into areas of
+interest.
+
+[discrete]
+[[search-overview]]
+== Search
+
+Filter for alerts, events, processes, and other important security data with the benefit of *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* in the  Search bar, which appears at the top of each page throughout the app. A date/time filter set to `Last 24 hours` is enabled by default, but can be changed to any time range. If you want to filter your search results with other fields, select **Add Filter,** followed by the field from which to filter and the operator (such `is not` or `is between`) for your query.
+
+To save specific filters and queries, click the *Save* button, then **Save current query.**
+
+
+[role="screenshot"]
+image::images/siem-click-swipe.png[]
+
+[discrete]
+[[page-tabs]]
+== Page tabs
+The {security-app} contains seven page tabs that enable analysts to view, analyze, and manage security data:
+* Overview
+* Detections
+* Hosts
+* Network
+* Timelines
+* Cases
+* Administration
+
+
+[float]
+[[overview-ui]]
+=== Overview page
+
+The Overview page provides a high-level snapshot view of detections, external alerts, and event trends. These trends are useful to assess overall system health and find anomalies that may require further investigation.
+
+image::images/overview-pg.png[]
+
+*Histograms*
+
+Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. You can select areas of interest in time-based histograms by selecting a region to reflect a date range, or by selecting a preset value in the timepicker. In the **Stack by** dropdown, you can select specific parameters for which to visualize the individual counts. For example, in the Detection alert trend histogram, stack by `signal.rule.name` to display the total counts by alert name within the specified time frame.
+
+TIP: All Elastic Security histograms, graphs, and tables contain an **Inspect** button so you can examine the {es} queries used to retrieve data throughout
+the app.
+
+Select the collapsable *Timeline* button on the rightmost side of the {security-app} to start an investigation. See <<timeline-ui-overview,>> for more information.
+
+*Host and network events*
+
+View event and host counts specific to Elastic data shippers and apps, such as **Auditbeats** or **Elastic Endpoint Security**. Expand each category to view specific counts of hosts or network events related to the selected category.
+
+[role="screenshot"]
+image::images/events-count.png[]
+
+
+[float]
+[[hosts-ui]]
+=== Hosts page
+
+The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline. See <<hosts-overview,Hosts page overview>> for more information.
```

Comment:
This hosts-overview page doesn't seem to exist in the book yet, so I omitted it for now:
```suggestion
The Hosts view provides key metrics regarding host-related security events, and a set of data tables that let you interact with the Timeline.
//See <<hosts-overview,Hosts page overview>> for more information.
```

Reply:
I think this change (omission of the hosts-overview link) is still necessary.